src/Normalizer/ModelNormalizer.php

```diff
@@ -8,9 +8,11 @@
 use SupportPal\ApiClient\Transformer\Transformer;
 use Symfony\Component\Serializer\Normalizer\ContextAwareDenormalizerInterface;
 use Symfony\Component\Serializer\Normalizer\ContextAwareNormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
 use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
 
-class ModelNormalizer implements ContextAwareNormalizerInterface, ContextAwareDenormalizerInterface
+class ModelNormalizer implements NormalizerInterface, DenormalizerInterface
```

Comment:
I think think this is necessary. The interfaces are only `@deprecated` (no deprecation notices?), and we need to continue to support PHP 7.2 because of WHMCS. 

```suggestion
class ModelNormalizer implements ContextAwareNormalizerInterface, ContextAwareDenormalizerInterface
```